modules/simple_forms_api/spec/fixtures/form_json/vba_21_0972-min.json

```diff
@@ -0,0 +1,57 @@
+{
+    "action": "submit",
+    "claimant_identification": "VETERAN",
+    "controller": "simple_forms_api/v1/uploads",
+    "form_number": "21-0972",
+    "format": "json",
+    "preparer_address": {
+        "city": "Anytown",
+        "country": "USA",
+        "postal_code": "12345",
+        "state": "CA",
+        "street": "124 Main St"
+    },
+    "preparer_full_name": {
+        "first": "Arthur",
+        "last": "Preparer"
+    },
+    "preparer_phone": "9876054321",
+    "preparer_qualifications": "Court-appointed representative",
+    "preparer_signing_reason": "They’re under 18 years old.",
+    "statement_of_truth_certified": true,
+    "statement_of_truth_signature": "Arthur Preparer",
+    "upload": {
```

Comment:
where did this upload layer come from? I don't see this in my submission from the frontend

Reply:
This is from the rails params. I can remove it since I think it's all duplicate anyway.